tools/appveyor.psm1

```diff
@@ -95,9 +95,9 @@ Function Test-DailyBuild
         return $true
     }
 
-    # if [Feature] is in the commit message,
+    # if [Feature] or [Package] is in the commit message,
     # Run Daily tests
-    if($env:APPVEYOR_REPO_COMMIT_MESSAGE -match '\[feature\]')
+    if($env:APPVEYOR_REPO_COMMIT_MESSAGE -match '(\[feature\])|(\[package\])')
```

Comment:
We don't need the change in AppVeyor. AppVeyor PR CI always generate packages.
Example: https://ci.appveyor.com/project/PowerShell/powershell/build/6.0.0-beta.9-6723

Reply:
got it, will remove